javalanglib/src/main/scala/java/lang/System.scala

```diff
@@ -6,7 +6,35 @@ import scala.scalajs.js
 import js.Dynamic.global
 import scala.scalajs.LinkingInfo.assumingES6
 
+import java.{util => ju}
+import scalajs.runtime.environmentInfo
+
 object System {
+
+  private[this] var systemProperties = loadSystemProperties()
+
+  private def loadSystemProperties(): ju.Properties = {
+    val sysProp = new ju.Properties()
+    sysProp.setProperty("java.version", "1.8")
+    sysProp.setProperty("java.vm.specification.version", "1.8")
+    sysProp.setProperty("java.vm.specification.vendor", "Oracle Corporation")
+    sysProp.setProperty("java.vm.specification.name", "Java Virtual Machine Specification")
```

Comment:
And if we are not running with sbt the version is just automagically gone? Probably not a good idea. I think we should propagate the version through the linking infos.